extra/intl-extension/composer.json

```diff
@@ -0,0 +1,39 @@
+{
+    "name": "twig/intl-extension",
+    "type": "library",
+    "description": "A Twig extension for Intl",
+    "keywords": ["twig", "intl"],
+    "homepage": "https://twig.symfony.com",
+    "license": "MIT",
+    "authors": [
+        {
+            "name": "Fabien Potencier",
+            "email": "fabien@symfony.com",
+            "homepage": "http://fabien.potencier.org",
+            "role": "Lead Developer"
+        }
+    ],
+    "require": {
```

Comment:
don't forget symfony/intl and ext-intl here ;)

Reply:
I've moved symfony/intl from require-dev to require, but I don't want to add ext-intl as one can use the polyfill.